setup.cfg

```diff
@@ -1,2 +1,8 @@
 [upload_docs]
 upload-dir = docs/_build/html
+
+[flake8]
+ignore =
```

Comment:
Globally disabling linter rules won't be accepted. If it's really necessary for a line, disable it just for that line.

Especially E241 and E305 which are common whitespace rules.

Reply:
E242 - 18 violations in __init__.py
E305 - 1 violation in __init__.py

I may as well fix these violations but did not want to clutter the commit with whitespace changes. For now I will revert this file.